recipe/meta.yaml

```diff
@@ -10,11 +10,11 @@ package:
 
 source:
   url: https://github.com/openucx/ucx/archive/v{{ ucx_version }}.tar.gz
-  sha256: c1741f2e08374b033ea82382c6ec9c4f73ede31205c107386648c098545a003d
+  sha256: e4ac6561d59653a738f0b73b20f946dd05fa2288ea954659f4a7b6020f994444
 
 build:
   skip: true  # [not linux]
-  number: 0
+  number: 1
```

Comment:
If we're bumping the version we don't need to bump the build number

Reply:
This was an oversight, this PR was only changing to release with no bump on the UCX version, so this was left by accident. We also have the `number` variable above, is there a reason we can't use it here @jakirkham ? It is currently only used by `outputs.name=ucx.build.number`, wondering whether we can use for both.

Reply:
Fixed in https://github.com/conda-forge/ucx-split-feedstock/pull/111/commits/50a5dc2ffb24b62fd1117a5fdde47b291b175472 .